### ISSUE TITLE
Child views

### ISSUE THREAD

**Issue by javichi** [pec1985/TiDraggable#4]
Not sure if this is a bug or it is the expected behavior...

I can not drag a draggable view if touch starts on a child view (label, text field, etc).

Thanks for your help

**Comment by rdisanto** [comment]
+1

**Comment by kosso** [comment]
How about if you add bubbleParent:false to the other views?

**Comment by pec1985** [comment]
Yeah... I have to figure that out. Definitely on my list of things to do.

**Comment by MisterEvilGuy** [comment]
I've created a big view called the Grid which is a container for the views that I can create called Elements. These elements are Draggable and also a child view of the Grid so I'm experiencing the same problem, except that my app crashes on Android 2.3.3 and 4.0

**Comment by vpsouza** [comment]
on your child itens, add this attribute:
touchEnabled:false

It works for me (a view with 2 childs: 1 image and 1 label)

**Comment by darknos** [comment]
the first thank you for excellent module. 

I tried to create sliding menu like Facebook and met the same problem with android - if I add any child to draggable view I can't drag the view. :( Could you look at this issue?

btw, it works prefect with subviews in iOS.

**Comment by caspahouzer** [comment]
Any news on adding dragviews as childs? I need this module so much on android!!
